src/components/carousel/MainCarousel.jsx

```diff
@@ -3,28 +3,68 @@ import { useEffect, useState } from 'react';
 import CarouselCards from './shared-components/CarouselCards';
 import CarouselTextbox from './shared-components/CarouselTextbox';
 
+
+
 const MainCarousel = ({ images }) => {
   const [current, setCurrent] = useState(0);
   const [autoPlay, setAutoPlay] = useState(true);
+  
 
   let timeOut = null;
 
   const paginationDots = images.map((img, index) => <div className={index === current ? 'pagination_dot pagination_dot-active' : 'pagination_dot'} onClick={() => setCurrent(index) && clearTimeout(timeOut)}></div>);
 
   useEffect(
     () => {
-      timeOut =
+     let timeOut =
```

Comment:
I think this could cause some issues, just something that Imran mentioned it may be worth checking in with him to get any suggestions